src/utils/queue_builder.rs

```diff
@@ -48,21 +48,13 @@ pub fn queue_builder(
 
         if status.count() >= 10 {
             // periodically, we need to flush our caches and ping the hubs
-            debug!("10 builds in a row; flushing caches");
+            debug!("10 builds in a row; pinging pubsubhubhub");
             status = BuilderState::QueueInProgress(0);
 
             match pubsubhubbub::ping_hubs() {
```

Comment:
I didn't know we had this 😆 do we know anyone using this? I'd rather have #809 I think.

Reply:
Dunno why it exists, but let's not remove it for now until we get more insights on who's using it.